website/dbt-versions.js

```diff
@@ -0,0 +1,20 @@
+exports.versions = [
+  // {
+  //   version: "1.1",
+  //   EOLDate: "2023-03-18"
+  // },
+  {
+    version: "1.0",
+    EOLDate: "2023-12-03"
+  },
+  {
+    version: "0.21",
+    EOLDate: "2022-06-30"
+  },
+  {
+    version: "0.20",
+    EOLDate: "2022-06-30"
```

Comment:
This feels like a fine middle ground for now. @runleonarun and I discussed this at length in our last 1:1.

My main concerns are:
- These versions have already reached EOL from a Core maintenance perspectively, but they'll continue to be supported in Cloud until June 30.
- The selection of v0.20 or v0.21 from the version dropdown implies that the user will see docs that are fully relevant for v0.20 or v0.21, which isn't quite true. We'll add some language to the `Core Versions` page to reflect this: [EOLDate](https://github.com/dbt-labs/docs.getdbt.com/issues/1217)

Ideally, we'd have a banner that we can show users for the next 3 months, while v0.20 + v0.21 are special exceptions. I don't think that should block shipping all the good work that's gone into this feature, and which we badly need to unblock other initiatives.

Reply:
For sure! This initial release will be minimum as far as actual versioned content goes. The only versioned content this PR currently includes come from the Node Selector > Commands pages, converting the tabs to the versionBlock component. 

I can imagine the more versioned content we add, the more we will want to adjust. This is a great item to add to our list for continuous improvements on this project.